tests/console/procps.pm

```diff
@@ -0,0 +1,38 @@
+use base "consoletest";
```

Comment:
How about using virtio console feature instead of consoletest?
Output is in text instead of bitmap, see e3fc5d807a1abe47198ba902550b00d43773c8e3
http://quasar.suse.cz/tests/1598#step/qa_test_klp/6
https://openqa.suse.de/tests/2376546/file/serial_terminal.txt

https://openqa.suse.de/tests/2376666#step/abort01/7
https://openqa.suse.de/tests/2376666/file/serial_terminal.txt

Reply:
interesting! Thanks for tip, I'll change the code :+1: